session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -1231,28 +1210,6 @@ public void insertAlignedRecordsOfOneDevice(
       List<List<TSDataType>> typesList,
       List<List<Object>> valuesList)
       throws IoTDBConnectionException, StatementExecutionException {
```

Comment:
two overload function which only differs in `haveSorted param`, thus merged to one